lib/presentation/widgets/custom_box_card.dart

```diff
@@ -0,0 +1,269 @@
+import 'package:cherrypic/core/constants/color.dart';
+import 'package:flutter/material.dart';
+
+import '../../core/constants/font.dart';
+
+class CustomBoxCard extends StatelessWidget {
+  final bool selected;
+  final Color borderColor;
+  final Color dotColor;
+  final String title;
+  final String subtitle;
+  final String priceText;
+  final Color? priceColor;
+  final VoidCallback onTap;
+  final Color? divideColor;
+
+  const CustomBoxCard({
+    super.key,
+    // required this.selected,
+    this.selected = false,
+    required this.borderColor,
+    required this.dotColor,
+    required this.title,
+    required this.subtitle,
+    required this.priceText,
+    required this.priceColor,
+    required this.onTap,
+    required this.divideColor,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    final screenWidth = MediaQuery.of(context).size.width;
+
+    return GestureDetector(
+      child: SizedBox(
+        width: screenWidth - 40,
+        child: Container(
+          padding: const EdgeInsets.all(16),
+          // margin: const EdgeInsets.symmetric(vertical: 6),
+          decoration: BoxDecoration(
+            border: Border.all(
+              color: selected ? borderColor : Colors.grey.shade300,
+              width: 2,
+            ),
+            borderRadius: BorderRadius.circular(12),
+            color: Colors.white,
+          ),
+          child: Row(
+            crossAxisAlignment: CrossAxisAlignment.center,
+            children: [
+              Expanded(
+                child: Column(
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  children: [
+                    Text(
+                      title,
+                      style: AppFont.size16.copyWith(
+                        fontWeight: FontWeight.bold,
+                      ),
+                    ),
+                    const SizedBox(height: 4),
+                    Text(subtitle, style: TextStyle(color: AppColor.subGrey)),
+                    selected
+                        ? Container(
+                            margin: const EdgeInsets.symmetric(vertical: 12),
+                            width: double.infinity,
+                            height: 1,
+                            child: CustomPaint(
+                              painter: DottedLinePainter(divideColor!),
+                            ),
+                          )
+                        : Container(
+                            margin: const EdgeInsets.symmetric(vertical: 12),
+                            width: double.infinity,
+                            height: 1,
+                            child: CustomPaint(
+                              painter: DottedLinePainter(Colors.white),
+                            ),
+                          ),
```

Comment:
```suggestion
Container(
  margin: const EdgeInsets.symmetric(vertical: 12),
  width: double.infinity,
  height: 1,
  child: CustomPaint(
    painter: DottedLinePainter(selected ? divideColor! : Colors.white),
  ),
),
```

다른 설정이 비슷해서 이렇게 합쳐도 좋을 것 같아요!

Reply:
아 맞네요. 색상만 달라지는거라 3항 연산자로 작성이 훨씬 좋아보이네요 수정하겠습니다

Reply:
수정완료했습니다